packages/aws-cdk-lib/aws-lakeformation/README.md

```diff
@@ -25,3 +25,87 @@ For more information on the resources and properties available for this service,
 (Read the [CDK Contributing Guide](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and submit an RFC if you are interested in contributing to this construct library.)
 
 <!--END CFNONLY DISCLAIMER-->
+
+### Example
+
+Using the lakeformation L1s is not always the most straight forward. Here is an example of creating a glue table and putting lakeformation tags on it. Note: this example uses deprecated constructs and overly permissive IAM roles. This example is meant to give a general idea of using the L1s; it is not production level.
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { Table, Database, DataFormat, Schema } from '@aws-cdk/aws-glue-alpha';
+import { CfnDataLakeSettings, CfnTag, CfnTagAssociation } from 'aws-cdk-lib/aws-lakeformation';
+
+export class LakeFormationTaggingStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const accountId = process.env.CDK_DEFAULT_ACCOUNT!;
+    const region = process.env.CDK_DEFAULT_REGION!;
```

Comment:
We can probably remove these and use `this.account` and `this.region` everywhere